sass/components/_icons.scss

```diff
@@ -0,0 +1,17 @@
+//Ikony potraktowałem póki co jako osobny komponent, w przyszłości trzeba będzie się zastanowić :) - Tomek
+
+.icon {
+    float: left;
+    position: relative;
+    left: 100px;
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-width: 50px;
+    min-height: 50px;
+    &--fame {
+        background-image: url("../static/assets/ui/IconHallOfFame.png");
+    }
+    &--rules {
+        background-image: url("../static/assets/ui/IconRules.png");
```

Comment:
Nie wiem czy nie lepiej byłoby po prostu ustawiać wartośc background-image w JSie w tym przypadku. Ale jak uważacie.

Reply:
Jeśli ma być to "icon" to nie może być to ta nazwa. Raczej "btnIcon" (lub buttonIcon" lub "textIcon". 
Osobiście "icon" zarezerwowałbym dla ikony głównej projektu- jeśli taka się pojawi.

To wyżej tylko moja opinia, ale to @nowakprojects może naprostuje, gdyż mogę się mylić.

Dodatkowo dodałbym to jako <i> a nie obrazek. Ale to też tylko moja, może mylna, opinia